context/opencensus-context/README.rst

```diff
@@ -5,3 +5,67 @@ OpenCensus Runtime Context
 
 .. |pypi| image:: https://badge.fury.io/py/opencensus-context.svg
    :target: https://pypi.org/project/opencensus-context/
+
+The **OpenCensus Runtime Context** provides the in-process context propagation.
+By default, thread local storage is used for Python 2.7, 3.4 and 3.5;
+contextvars is used for Python >= 3.6, which provides asyncio support.
+
+Installation
+------------
+
+This library comes by default when you install OpenCensus, there is no need
+to install it explicitly.
+
+Usage
+-----
+
+By default, context would flow in the same thread and async task. There are
```

Comment:
"flow in the same thread and async task" is confusing here

Reply:
I'm trying to improve the wording here, please let me know if it is stilling confusing. Thanks.